products/accesscontextmanager/terraform.yaml

```
@@ -14,6 +14,13 @@
 --- !ruby/object:Provider::Terraform::Config
 overrides: !ruby/object:Overrides::ResourceOverrides
   AccessPolicy: !ruby/object:Overrides::Terraform::ResourceOverride
+    docs: !ruby/object:Provider::Terraform::Docs
+      warning: |
+        If you are using User ADCs (Application Default Credentials) with this resource,
+        you must specify a `billing_project` and set `user_project_override` to true 
+        in the provider configuration. Otherwise ACM API will returns 403 error. 
```

nitpick: small wording correction here + the other doc comments if you don't mind.

```suggestion
        in the provider configuration. Otherwise the ACM API will return a 403 error. 
```

---

fixed